src/async_impl/response.rs

```diff
@@ -71,6 +72,18 @@ impl Response {
         &mut self.body
     }
 
+    /// Get the `Version` of this `Response`.
+    #[inline]
+    pub fn version(&self) -> &Version {
+        &self.version
+    }
+
+    /// Get a mutable reference to the `Version` of this `Response`.
+    #[inline]
+    pub fn version_mut(&mut self) -> &mut Version {
```

Comment:
Seeing as this is a client library, is there value in being able to mutate the received version? I think all that is needed is just a getter, right?